pydocstringformatter/utils/argument_parsing.py

```diff
@@ -80,3 +99,37 @@ def _parse_toml_file(
             arguments += _parse_toml_option(key, value)
 
         parser.parse_args(arguments, namespace)
+
+
+def _load_formatters_default_option(
+    parser: argparse.ArgumentParser,
+    namespace: argparse.Namespace,
+    formatters: List[Formatter],
+) -> None:
+    """Parse the state of a the list of formatters based on their 'optional' attribute."""
```

Comment:
```suggestion
    """Parse the state of a list of formatters based on their 'optional' attribute."""
```

Reply:
Had to apply that locally anyway because the docstring was too long for flake8 :)